location/templates/claim.html

```diff
@@ -3,7 +3,13 @@
 {% block breadcrumbs %}
 <a href="{% url 'search' %}">Search</a>
 <a href="{% url 'location' pk=apt.location.id %}">{{ apt.location.full_address }}</a>
-<a href="{% url 'apartment' pk=apt.location.id suite_num=apt.suite_num %}">{{ apt.suite_num }}</a>
+<a href="{% url 'apartment' pk=apt.location.id apk=apt.id %}">
+    {% if apt.suite_num %}
+    {{ apt.suite_num }}
+    {% else %}
+    1
```

Comment:
This should be also changed to `#`?

Reply:
good catch, I felt like I was missing it somewhere.